### DISCUSSION TITLE
Google Home vs Home Assistant

### DISCUSSION THREAD

**Discussion by dwarfish** [indomus/forum#3219] [HUB: Home Assistant]
Ciao a tutti,
è da diversi mesi che gli speaker di Google mi rispondo di non poter accedere ai dispositivi esposti da Home Assistant tramite progetto su GCP: aprendo l'app Home li trovo grigi e non disponibili mentre da HA tutto funziona regolarmente. La problematica sembra del tutto randomica infatti così come appare, scompare. Ho provato a forzare la sincronizzazione dei dispositivi ogni 30 minuti, oltre che al riavvio di HA, ma non ho risolto. Nel tentativo estremo di risolvere il problema, ho scollegato l'account di HA dall'app di Google Home ma ora, ogni volta che tento di ricollegarlo, mi appare l'errore "Impossibile raggiungere [test] casa. Riprova." Ho creato anche un nuovo progetto su GPC e riconfigurato tutto su HA ma, anche in questo caso, mi da lo stesso errore. Risultato ora non ho più i dispositivi visibili su Google Home.

Al momento HA è installato come SO su un Raspberry Pi 3:
```
  OS Version:               Home Assistant OS 13.1
  Home Assistant Core:      2024.8.3
```
Per la creazione dei certificati HTTPS utilizzo Let's Encrypt tramite il componente Duck DNS che utilizzo anche per l'FQDN. Non ci sono FW o reverse proxy in quanto HA è esposto su interner tramite un semplice Port Forwarding del router Fritz!Box.

Idee su come risolvere il problema?

Grazie,
^SiMoNe^

**Comment by Scialla** [comment]
Utilizzi la configurazione peggiore per esporre HA su internet.... se non hai voglia di spendere sti 10€/anno per un dominio e utilizzare un tunnel cloudflare, almeno usa un reverse proxy (c'è la guida sul sito per caddy) e lascia gestire a lui i certificati e, assolutamente, cambia ddns, abbandona Duckdns, fonte di innumerevoli problemi, e passa ad altro, ad esempio no-ip

**Reply by dwarfish** [reply]
Grazie @Scialla per i suggerimenti, vediamo se riesco a mettere il reverse proxy senza dover rifare tutta la configurazione. Intanto comincio a passare su no-ip e ti faccio sapere

Ciao,
^SiMoNe^

**Reply by dwarfish** [reply]
Riconfigurato HA da zero con no-ip e Nginx, rifatto il progetto su GCP e ricollegato il tutto sull'app di Google Home senza, per adesso, problemi!

Grazie @Scialla
